gnome-base/gnome-settings-daemon/gnome-settings-daemon-46.0-r1.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{11..13} )
```

Comment:
I think we should keep 3.10 support (until there are more concrete dates/pkgcheck warnings for removal), otherwise we can cause breakage on reverse deps such as:
https://bugs.gentoo.org/936437